pkg/virt-config/config-map.go

```diff
@@ -369,6 +371,14 @@ func setConfigFromConfigMap(config *v1.KubeVirtConfiguration, configMap *k8sv1.C
 		}
 	}
 
+	if pvcReserve := strings.TrimSpace(configMap.Data[MinimumReservePVCBytesKey]); pvcReserve != "" {
```

Comment:
Since the configmap approach is deprecated, we don't add new tunables to it. Do you need it?

Reply:
We haven't used the KubeVirt CR, since we don't use virt-operator, but I think that can be remedied without too much effort. Especially if the goal is to deprecate it.

The main reason I have these is because the existing test patterns  in template_test.go use testutils.UpdateFakeClusterConfig which uses a config map and constants in the virt-config to update the KubeVirt CR. I can probably fix that as well if needed.

Reply:
> We haven't used the KubeVirt CR, since we don't use virt-operator, but I think that can be remedied without too much effort. Especially if the goal is to deprecate it.

FYI you don't need to use the operator to use the KubeVirt CR. Just register the CRD and opst the configuration. Once you have it in place, delete the config-map and it will be the new config-source for all components.

> The main reason I have these is because the existing test patterns in template_test.go use testutils.UpdateFakeClusterConfig which uses a config map and constants in the virt-config to update the KubeVirt CR. I can probably fix that as well if needed.

Yes that  should be pretty easy to fix, if you use `testutils.UpdateFakeKubeVirtClusterConfig` instead.

Reply:
Ok, I refactored the rest of template_test.go as well to keep things tidy and consistent. Hopefully we don't need to continue testing with the configmap on all of the others!

Reply:
> > We haven't used the KubeVirt CR, since we don't use virt-operator, but I think that can be remedied without too much effort. Especially if the goal is to deprecate it.
> 
> FYI you don't need to use the operator to use the KubeVirt CR. Just register the CRD and opst the configuration. Once you have it in place, delete the config-map and it will be the new config-source for all components.
> 
Yes, I understand. It's just that initially there was no reason to apply the KubeVirt CR since we weren't using virt-operator.

> > The main reason I have these is because the existing test patterns in template_test.go use testutils.UpdateFakeClusterConfig which uses a config map and constants in the virt-config to update the KubeVirt CR. I can probably fix that as well if needed.
> 
> Yes that should be pretty easy to fix, if you use `testutils.UpdateFakeKubeVirtClusterConfig` instead.

Ok, I refactored the rest of template_test.go as well to keep things tidy and consistent. Hopefully we don't need to continue testing with the configmap on all of the others!